assets/src/edit-story/components/dropDown/index.js

```diff
@@ -289,10 +295,11 @@ function DropDown({ options, value, onChange, disabled, ariaLabel }) {
   };
 
   useEffect(() => {
-    if (focusedValue) {
+    if (!isNullOrUndefined(focusedValue)) {
       const findIndex = options.findIndex(
         (item) =>
-          focusedValue && item.value.toString() === focusedValue.toString()
+          !isNullOrUndefined(focusedValue) &&
```

Comment:
This is already checked above.

Reply:
Yes, replaced by using foundIndex usememo